cmake/onnxruntime_unittests.cmake

```diff
@@ -677,8 +678,9 @@ if (WIN32)
 endif()
 
 if (onnxruntime_BUILD_SHARED_LIB)
-  set(onnxruntime_perf_test_libs onnxruntime_test_utils onnx_test_runner_common onnxruntime_common re2::re2
-          onnx_test_data_proto onnx_proto ${PROTOBUF_LIB} ${GETOPT_LIB_WIDE} onnxruntime ${SYS_PATH_LIB}
+  set(onnxruntime_perf_test_libs onnx_test_runner_common onnxruntime_test_utils onnxruntime_framework onnxruntime_util 
+          onnxruntime_graph onnxruntime_common onnxruntime_mlas onnxruntime re2::re2
```

Comment:
No, it's wrong. Your build won't pass. 

The original design is: the onnxruntime_perf_test tool could either dynamically link to onnxruntime libs or static link to them. But shouldn't do both of them at the same time, it doesn't make sense and it may cause problem while the linker is doing symbol resolving.  So I intentionally didn't put ONNX, onnxruntime_graph and onnxruntime_framework at there. onnxruntime_common is tiny, stateless and it doesn't have any ONNX special logical so it's safe to have it here. But not the others.

If you still insist on doing this,  there is another thing you must pay attention: [The order of the library names matters](https://github.com/microsoft/onnxruntime/blob/master/docs/cmake_guideline.md#static-library-order-matters). You did it wrong it the build would never pass. 

Please take a look at the high level design of onnxruntime below:
![img](https://github.com/microsoft/onnxruntime/raw/master/docs/onnxruntime_dependencies.png)

onnxruntime_util depends on onnxruntime_framework. So it should be on the left side of onnxruntime_framework